test/ui-testing/publishButton.test.js

```diff
@@ -9,14 +9,14 @@ describe('Publish button', () => {
   
   test('Publish button is enabled', async () => {
     // Check initially that Publish button is disabled.    
-    expect(await page.evaluate(() => document.querySelector('.ple-textarea').disabled)).toBe(true);
+    expect(await page.evaluate(() => document.querySelector('.ple-publish').disabled)).toBe(true);
     // Add title.
     await page.evaluate(() => {
       document.querySelector('.ple-module-title input').value = 'A title';
       document.querySelector('.ple-module-title input').dispatchEvent(new KeyboardEvent('keydown', { "code": "9" })); // random key
     });
     // Check final state of Publish button.
-    expect(await page.evaluate(() => document.querySelector('.ple-textarea').disabled)).toBe(false);
+    expect(await page.evaluate(() => document.querySelector('.ple-publish').disabled)).toBe(false);
   });
```

Comment:
Hi @jywarren I noticed you didn't add the `timeout` variable here. Is this part of the new change or is it something you missed out?

Reply:
I think I was able to set it globally on the first few lines and so didn't need to set it per-test.

Reply:
Ah, interesting, actually on checking, I am not setting the timeout. I think that means we could remove that line, actually. 

For reference, I'm stuck on a related issue (going past the timeout limit in this case) in this test https://github.com/publiclab/image-sequencer/pull/2026

https://github.com/publiclab/image-sequencer/blob/58eb3ef6b1caba88aaf217d93ec085a14765d8aa/test/ui-2/test/OverlayInput.test.js#L1-L12

That actually is globally setting timeout. But here I think i just removed it and for the purposes of our test it wasn't needed. I think the default timeout is 5 or 10 seconds? So i guess that's enough for this.